### PR TITLE
Updated windows support

### PR DESCRIPTION
**Pull request by krjensen**
Consul is now able to run as a native windows server, so I have removed the need for NSSM and maked the code more clean.

### CODE REVIEW

File: manifests/install.pp [hunk 1]
```diff
@@ -9,7 +9,7 @@
       ensure => 'directory',
       owner  => $consul::user_real,
       group  => $consul::group_real,
-      mode   => '0755',
+      mode   => $consul::data_dir_mode,
```
Comment (krjensen) [inline manifests/install.pp]: On windows the group needs "full controle" on the data_dir

File: manifests/params.pp [hunk 2]
```diff
@@ -57,13 +57,15 @@
...
   case $facts['os']['name'] {
     'windows': {
-      $binary_group = 'Administrators'
-      $binary_mode = '0755'
+      $data_dir_mode = '0775'
+      $binary_group = undef
+      $binary_mode = '0775'
       $binary_name = 'consul.exe'
-      $binary_owner = 'Administrator'
+      $binary_owner = 'NT AUTHORITY\NETWORK SERVICE'
```
Comment (krjensen) [inline manifests/params.pp]: It's more correct to run the service as NETWORK SERVICE and not the local Administrator.

**Comment by krjensen** [comment]
> HI @krjensen, thank for the PR. Can you document the limitations in the README.md? I restarted the failed travis job. Please also have a look at the used email address in the commits. It isn't associated with your github account.

Hi @bastelfreak. I have updated the README.md and I have associated my work email with me private github user, so this should be correct now.

**Comment by bastelfreak** [comment]
Hi,
the travis issues are caused by https://github.com/puppetlabs/beaker-puppet/pull/102. We need to wait for a new release of beaker-puppet.

**Pull request merged**